from authx.errors.social import SocialException

__all__ = ["SocialException"]
